### DISCUSSION TITLE
guide/%E9%A1%B9%E7%9B%AE%E5%AE%9E%E6%88%98/%E5%BE%AE%E4%BF%A1%E5%B0%8F%E7%A8%8B%E5%BA%8FAI

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#803] [Announcements]
# guide/%E9%A1%B9%E7%9B%AE%E5%AE%9E%E6%88%98/%E5%BE%AE%E4%BF%A1%E5%B0%8F%E7%A8%8B%E5%BA%8FAI

功能：根据用户输入的选项，随机选择一个选项。用户点击新建按钮，新建选项，并在选项按钮前给出序号，用户点击生成选择按钮，随机选择一个选项并显示在下方。用户点击清空按钮清空已生成的随机选择，用户点击新建选择，清空所有选项和已生成的随机选择 用途：例如，当您有多个选择，但不知道如何选择时，可以使用此功能。要在 uni-app 中设计一个 Vue 文件，同时遵...

http://localhost:8080/guide/%E9%A1%B9%E7%9B%AE%E5%AE%9E%E6%88%98/%E5%BE%AE%E4%BF%A1%E5%B0%8F%E7%A8%8B%E5%BA%8FAI.html

**Comment by virtualfounder** [comment]
大萨达
